java/Anton-Kozyk/src/com/zoolatech/lecture4/tasks/_7/Task7.java

```diff
@@ -0,0 +1,37 @@
+package com.zoolatech.lecture4.tasks._7;
+
+/**
+ * Create a method that accepts a string of digits and after each
+ * digit appends its string representation in parentheses
+ *
+ * Input: “12345”
+ * Output: “1(one)2(two)3(three)4(four)5(five)”
+ */
+
+public class Task7 {
+    public static void main(String[] args) {
+        System.out.println(digitToString("12345"));
+        System.out.println(digitToString("56789"));
+    }
+
+    static String digitToString(String str) {
+        StringBuilder result = new StringBuilder();
+
+        for (char ch : str.toCharArray()) {
+            switch (ch) {
+                case '1' -> result.append("1(one)");
```

Comment:
try to improve your code with removing a duplication of `result.append` operation